save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/vulnerabilities/Vulnerability.kt

```diff
@@ -97,5 +101,7 @@ class Vulnerability(
         dates = emptyList(),
         participants = emptyList(),
         status = status,
+        creationDateTime = createDate?.toKotlinLocalDateTime(),
```

Comment:
can we change field name in entity and database to `createdDate`\`created` and `updatedDate`\`lastUpdatedDate`\`lastUpdated` accordingly ?

Reply:
We will have to change a lot of tables. Update and Create Date are BaseEntity fields.

Reply:
I see: `com.saveourtool.save.spring.entity.BaseEntityWithDate` will create an issue in backlog for it

Reply:
> We will have to change a lot of tables. Update and Create Date are BaseEntity fields.

©Cheshiriks